models/Version/ElementDescriptor.php

```diff
@@ -74,4 +76,16 @@ public function setId(int $id)
 
         return $this;
     }
+
+    /**
+     * get the cache tag for the element
+     *
+     * @return string
+     */
+    public function getCacheTag()
```

Comment:
Maybe we should call this `getCacheKey()` , see `\Pimcore\Model\Element\AbstractElement::getCacheKey`
What do you think? I guess this would be a bit more specific to it's use case.